src/errors/runtime/templates.js

```diff
@@ -102,5 +102,9 @@ export default {
     [RUNTIME_ERRORS.unexpectedIPCBodyPacket]:                            'Cannot create an IPC message due to an unexpected IPC body packet.',
     [RUNTIME_ERRORS.unexpectedIPCTailPacket]:                            'Cannot create an IPC message due to an unexpected IPC tail packet.',
     [RUNTIME_ERRORS.cannotUseAllowMultipleWindowsOptionForLegacyTests]:  'You cannot run Legacy API tests in multi-window mode.',
-    [RUNTIME_ERRORS.cannotUseAllowMultipleWindowsOptionForSomeBrowsers]: 'You cannot use multi-window mode in {browsers}.'
+    [RUNTIME_ERRORS.cannotUseAllowMultipleWindowsOptionForSomeBrowsers]: 'You cannot use multi-window mode in {browsers}.',
+    [RUNTIME_ERRORS.cannotRunLocalNonHeadlessBrowserWithoutDisplay]:     'You run {browserAlias} browser with graphic interface in Linux without graphic subsystem. ' +
```

Comment:
AFAIR we discussed that we should suggest using `chrome:headless`, `chromium:headless` or `firefox:headless` in this case.

Reply:
@VasilyStrelyaev, can you take a look at this warning message?

> You run "{browserAlias}" browser with graphic interface in Linux without graphic subsystem. Try to run in headless mode. For more information see https://devexpress.github.io/testcafe/documentation/guides/concepts/browsers.html#test-in-headless-mode

Reply:
What about:

```js
    [RUNTIME_ERRORS.cannotRunLocalNonHeadlessBrowserWithoutDisplay]:     'Your Linux version does not have a graphic subsystem to run {browserAlias} with a GUI. You can launch {browserAlias} in headless mode (use \'{browserAlias}:headless\' as the browser alias). For more information, see {link}.'
```

(checked with Dirk)

Reply:
But there is a problem with this approach. When the user specify a browser by `path`:
```
testcafe "path:/usr/bin/chromium" test.js
```
`browserAlias` will be equal to `path:/usr/bin/firefox` and the following error message will appear:

> Your Linux version does not have a graphic subsystem to run path:/usr/bin/firefox with a GUI. You can launch path:/usr/bin/firefox in headless mode (use **'path:/usr/bin/firefox:headless'** as the browser alias). For more information, see https://devexpress.github.io/testcafe/documentation/guides/concepts/browsers.html#test-in-headless-mode'

As described in the link above, the user should use the browser alias instead of the `path:` prefix. For example:
```
testcafe "firefox:/usr/bin/firefox:headless" test.js
```

Thus, just as a suggestion, I propose:

> Your Linux version does not have a graphic subsystem to run '{browserAlias}' with a GUI. You can launch the browser in headless mode. If you use a portable version of the browser, then use the browser alias instead of the 'path' prefix. For more information, see {link}.

By the way, are there any style-guides for using `'` instead of `"`? (may look inconsistent).

Reply:
Checked with Dirk v2:

```js
Your Linux version does not have a graphic subsystem to run {browserAlias} with a GUI. You can launch the browser in headless mode. If you use a portable browser version, specify the browser alias before the path instead of the 'path' prefix. For more information, see {link}.
```

Reply:
I always use single quotes, but probably we have double quotes somewhere. IDK about the standard though, I should ask I guess.